dunwoody_disaster/Item.py

```diff
@@ -30,6 +30,35 @@ def serialize(self) -> dict:
             "stamina": self.staminaCost,
         }
 
+    def preview_widget(self) -> QWidget:
```

Comment:
What is the point of having this and `widget()`?

Reply:
different sizing for different uses